src/content/en/tools/puppeteer/_toc.yaml

```diff
@@ -0,0 +1,30 @@
+toc:
+- heading: Get Started
+- title: Overview
+  path: /web/tools/puppeteer/
+- title: API Reference
+  path: /web/tools/puppeteer/api
```

Comment:
Instead of creating a real page for this, what about just linking directly to the GitHub docs pages. For example:

```
- title: API Reference
  style: accordion
  section:
  - title: v1.0 (latest)
    section:
    - title: API Docs
      path: https://github.com/GoogleChrome/puppeteer/blob/v1.0/docs/api.md
      status: external
    - title: Release Notes
      path: https://github.com/GoogleChrome/puppeteer/releases/tag/v1.0
      status: external
  - title: v0.13.0
    section:
    - title: API Docs
      path: https://github.com/GoogleChrome/puppeteer/blob/v0.13.0/docs/api.md
      status: external
    - title: Release Notes
      path: https://github.com/GoogleChrome/puppeteer/releases/tag/v0.13.0
      status: external
  - title: v0.12.0
    section:
    - title: API Docs
      path: https://github.com/GoogleChrome/puppeteer/blob/v0.12.0/docs/api.md
      status: external
    - title: Release Notes
      path: https://github.com/GoogleChrome/puppeteer/releases/tag/v0.12.0
      status: external
```

Reply:
I originally had that, but that but moved it over to a page b/c:
- that list is going to get bigger over time
- Eventually we'll move the API docs here. api.md will become a big boy page.

Reply:
Also trying to think about ways to not have to add new links ever time there's a release. Do you know what other projects are doing?

Reply:
Actually, the collapser is really nice. I'm going to do this for now.

Reply:
done

Reply:
@gauntface was adding a new item for each release I believe.